mailpoet/tests/acceptance/Subscriptions/WooCheckoutAutomateWooSubscriptionsCest.php

```diff
@@ -55,18 +55,6 @@ public function checkoutOptInEnabled(\AcceptanceTester $i, $scenario) {
     $i->dontSee(self::AUTOMATE_WOO_OPTIN_TEXT);
   }
 
-  public function checkoutOptInUnchecked(\AcceptanceTester $i) {
```

Comment:
Same as for the integration test, I think we should rather check the subscriber was not added instead of removing the tests.

Reply:
Done